<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>128</width>
    <height>128</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="6" column="0">
       <widget class="QListView" name="faceView">
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="resizeMode">
         <enum>QListView::Fixed</enum>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="startButton">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Find Imposter</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="scanButton">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
</string>
        </property>
        <property name="text">
         <string>Detect Faces</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="PlotWidget" name="graphWidget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout"/>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="statsButton">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <!-- <property name="text">
         <string>View Summary</string>
        </property> -->
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="stopButton">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
